README.md

```diff
@@ -0,0 +1,2 @@
+The deploy script will tag node with name equal to the content of the NODE_SELECTOR environment variable, and deploy nym mixmode to it.
```

Comment:
This file is outdated, we should either remove it or reflect here that we are using the load balancer ip

Reply:
deleted will create new once finished